modules/ppcp-button/src/Assets/SmartButton.php

```diff
@@ -1092,9 +1092,9 @@ private function load_button_component() : bool {
 		}
 		if (
 			$this->context() === 'product'
-			&& $this->settings->has( 'button_product_enabled' )
+			&& ( $this->settings->has( 'button_product_enabled' )
 			&& $this->settings->get( 'button_product_enabled' )
-			|| $this->settings->has( 'message_product_enabled' )
+			|| $this->settings->has( 'message_product_enabled' ) )
```

Comment:
I think it's missing `get`?

```
		if (
			$this->context() === 'product'
			&& (
				( $this->settings->has( 'button_product_enabled' ) && $this->settings->get( 'button_product_enabled' ) ) ||
				( $this->settings->has( 'message_product_enabled' ) && $this->settings->get( 'message_product_enabled' ) )
			)
		) {
```

Reply:
fixed a1454dbfd7e60928b6e1926c94c7f269b621f383